openhgnn/models/MAGNN.py

```diff
@@ -433,56 +476,98 @@ def mp_instance_sampler(g, metapath_list):
 
     Notes
     -----
-    Please make sure that the metapath in metapath_list are all symmetric
+    Please make sure that the metapath in metapath_list are all symmetric and bidirectional. If not bidirectional,
+    please make sure that the leftmost element in a metapath in the dst node type.
```

Comment:
What's the meaning of symmetric and bidirectional metapath?  Also `the leftmost element in a metapath in the dst node type` is broken.  I guess you meant `the starting node type of a metapath is the same as its ending node type`.

Reply:
Actually the metapath can only be symmetric so the starting node type and the ending node type must be the same. I'll delete useless comments later.

Reply:
So it must be something like "ABA" but never "ABCA"?  Just curious.